.storybook/decorators/with-rtl.js

```diff
@@ -16,15 +16,19 @@ export const WithRTL = (story, context) => {
     setup() {
       const element = ref()
       const { i18n } = useContext()
+      const setLanguage = (direction) => {
+        element.value.ownerDocument.documentElement.setAttribute(
+          'dir',
+          direction?.value ?? 'ltr'
+        )
+      }
```

Comment:
Instead of combining `watch` and `onMounted` you can just add `{ immedate: true }` as the third parameter to `watch` I think.

Reply:
I've been doing immediate watch wrong my whole life. Fixed in f06e44e.